indico/modules/events/contributions/templates/management/_contribution_list.html

```diff
@@ -216,4 +216,15 @@
             {%- endif %}
         {%- endcall %}
     {%- endif %}
+        <script>
+        {% if selected_entry %}
+            $('#search-input').val('#{{ selected_entry }}').trigger('change');
+        {% endif %}
+
+        setupContributionList({
```

Comment:
IIRC `setupContributionList` registers events delegate-style so reloading that list doesn't need to execute this code again. If it does I'd rather keep it in the old place and use `indico:htmlUpdated` events to run code that needs to be re-executed